.travis.yml

```diff
@@ -11,15 +11,15 @@ cache:
 env:
   global:
   matrix:
-    - MODULE='single'
-    - MODULE='multiple'
+    - MODULE='multiple' CATEGORY="com.amazon.titan.testcategory.MultipleItemTests"
+    - MODULE='single' CATEGORY="com.amazon.titan.testcategory.SingleItemTests"
 
 
 matrix:
   # https://docs.travis-ci.com/user/customizing-the-build#Rows-that-are-Allowed-to-Fail
   allow_failures:
-    - env: MODULE='single' CATEGORY="com.amazon.titan.testcategory.MultipleItemTests"
-    - env: MODULE='multiple' CATEGORY="com.amazon.titan.testcategory.SingleItemTests"
+    - env: MODULE='multiple' CATEGORY="com.amazon.titan.testcategory.MultipleItemTests"
```

Comment:
I guess the additional split builds will go here. Does the order matter? if not lets not change here